hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxySupport.java

```diff
@@ -116,10 +117,13 @@
 abstract class MapProxySupport extends AbstractDistributedObject<MapService> implements InitializingObject {
 
     protected static final String NULL_KEY_IS_NOT_ALLOWED = "Null key is not allowed!";
+
     protected static final String NULL_VALUE_IS_NOT_ALLOWED = "Null value is not allowed!";
     protected static final String NULL_PREDICATE_IS_NOT_ALLOWED = "Predicate should not be null!";
     protected static final String NULL_LISTENER_IS_NOT_ALLOWED = "Null listener is not allowed!";
 
+    private static final int CHECK_IF_LOADED_TIMEOUT = 60;
```

Comment:
Thx! Yes at first it was like this:
`FutureUtil.waitWithDeadline(singleton(loadingFuture), 60, SECONDS, logAllExceptions(WARNING));`
For me it's the most readable format, since I see 60 and seconds - but Checkstyle complained about matgic number, which doesn't make much sense in this context, but anyway.
I'll rename it to `CHECK_IF_LOADED_TIMEOUT_SECONDS`